api/v2/api_test.go

```diff
@@ -1832,6 +1833,51 @@ func Test_API_Initialize_Main_Network(t *testing.T) {
 	api.Close()
 }
 
+func Test_API_Initialize_ListenAndServe(t *testing.T) {
+	// load configuration
+	cfg, err := config.LoadConfig("../../testenv/config.json")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// setup fake mock clients
+	fakeLens := &mocks.FakeIndexerAPIClient{}
+	fakeOrch := &mocks.FakeServiceClient{}
+	fakeSigner := &mocks.FakeSignerClient{}
+
+	type args struct {
+		certFilePath string
+		keyFilePath  string
+	}
+	tests := []struct {
+		name string
+		args args
+	}{
+		{"NoTLS", args{"", ""}},
+		{"TLS", args{"../../testenv/certs/api.cert", "../../testenv/certs/api.key"}},
```

Comment:
should add a case for broken files (ie provide the README.md as an argument)

Reply:
Good idea